openreview/workflows/simple_dual_anonymous_workflow/process/venue_group_template_process.py

```diff
@@ -26,14 +26,27 @@ def process(client, edit, invitation):
         root_id = paths[1]       
     client.add_members_to_group('host', root_id)
 
+    workflow_invitations = [f'{venue_id}/-/Submission', f'{venue_id}/-/Submission/Dates', f'{venue_id}/-/Submission/Form_Fields', f'{venue_id}/-/Submission/Notifications', f'{venue_id}/-/Submission_Change_Before_Bidding',
```

Comment:
can we list only the invitation that represent the step? then Xukun can find the ones that we use to edit them.